### ISSUE TITLE
Mod request: Auto-hide second monitor taskbar.

### ISSUE THREAD

**Issue by mateusfn98** [ramensoftware/windhawk-mods#170]
I searched a lot and don't find any tweak to auto-hide taskbar only on the second monitor. Found many ppl asking this soluction on forums.
If possible, it's a very useful project.

**Comment by gregclerk** [comment]
I agree.  This would be great especially for all monitors - I have 6 and 2 are stacked vertically so I'd only like to auto-hide the taskbar on the top one.

**Comment by mateusfn98** [comment]
Still don't find a solution :(
Is there a way to do this with Windows 11 Taskbar Styler?
Someone know what is the property that changes the behaviour to auto-hide or not?

**Comment by m417z** [comment]
> Is there a way to do this with Windows 11 Taskbar Styler?

Probably not, you can use it to make the taskbar invisible, but it will still occupy the space when windows are maximized.

**Comment by bergamin** [comment]
@m417z, if you can style only the second monitor's taskbar with it, wouldn't it be possible to make the taskbar's width = 0 on the second monitor? I don't have this mod, so I don't really know...

**Comment by m417z** [comment]
@bergamin if hiding it is all you're after, why not use the option that Windows provides?

![image](https://github.com/user-attachments/assets/cf32c013-4a15-45b0-b0b9-36216707e224)

**Comment by bergamin** [comment]
There is no option for auto-hiding only the second monitor taskbar. Only to fully hide it. I honestly don't care about auto-hiding. I was trying to help the other person who asked for it. The vertical taskbar mod already solved the issue I wanted to solve for myself

**Comment by mateusfn98** [comment]
> @bergamin if hiding it is all you're after, why not use the option that Windows provides?
> 
> ![image](https://private-user-images.githubusercontent.com/4129781/371874545-cf32c013-4a15-45b0-b0b9-36216707e224.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyNjk5ODYsIm5iZiI6MTcyODI2OTY4NiwicGF0aCI6Ii80MTI5NzgxLzM3MTg3NDU0NS1jZjMyYzAxMy00YTE1LTQ1YjAtYjBiOS0zNjIxNjcwN2UyMjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDdUMDI1NDQ2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzNhNjQ5ZTYyODgxOWVmMzA1MzRlNDE4NzY4NmQzZjRmMzEwMzBkZTYzMmUwNjY2N2M1MGQyZTVjYTRlYTQzMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.jutAaN38KjW2A199L96Wgd5moEoP_xg2Dxjb8bKDpCE)

With this option, the system auto-hides all monitors, but I was wondering to only auto-hide the second monitor.

**Comment by ArmandBernard** [comment]
Commented on [here](https://ramensoftware.com/windhawk-mods-for-the-windows-11-taskbar/comment-page-3#comment-15712), but I'd also be interested in this feature.

Specifically so I can auto-hide the taskbar only on my main monitor, as it's an OLED and I want to avoid burn-in, but I still want to see taskbar progress, the time etc. by glancing at my other monitors.

**Comment by jirat-tangthawin** [comment]
I can auto-hide the taskbar only focus monitor, the other the taskbar will show. I was wondering will hide taskbar at all monitor when maximized.

**Comment by FleXi2108** [comment]
> > Commented on [here](https://ramensoftware.com/windhawk-mods-for-the-windows-11-taskbar/comment-page-3#comment-15712), but I'd also be interested in this feature.
> > Specifically so I can auto-hide the taskbar only on my main monitor, as it's an OLED and I want to avoid burn-in, but I still want to see taskbar progress, the time etc. by glancing at my other monitors.
> 
> Exactly the same use case for me. Avoid taskbar burn in on main monitor and always keep the clock on secondary taskbar. Would love to see that implemented !

Just got my OLED today, so I'm on the same boat.
Only wnat to hide it on the OLED monitor but still be able to see the clock and other stuff at a glance on my 2nd monitor

**Comment by mateusfn98** [comment]
> Also putting in a vote for having the taskbar autohide be a per monitor setting. Have 3 monitors, want to only autohide on one of them. I tested Display Fusion which will do it, but I don't need the overhead of that entire suite in order to have auto hide for just one monitor, otherwise I'm sure its a decent program for other windows tweaks that I don't particularly need.

Would really be very useful, because my apps shows in the taskbar where the app is open.
Still can't find a way to tweak this, if someone could help make it with "Windows 11 Taskbar Styler" or something like that.

**Comment by TroyB9** [comment]
Cool, testing now and it seems to work!   Had to muck with the monitor numbers that did not match windows display settings, but thats it so far.  Thanks !!
